### ISSUE TITLE
Frustum test failure

### ISSUE THREAD

**Issue by knelli2** [sxs-collaboration/spectre#4013]
# Bug reports:

https://github.com/sxs-collaboration/spectre/runs/6714495282?check_suite_focus=true

```
209/1325 Test  #299: Unit.Domain.CoordinateMaps.Frustum ..........................................***Failed    2.18 sec
Charm++> No provisioning arguments specified. Running with a single PE.
         Use +auto-provision to fully subscribe resources or +p1 to silence this message.
Charm++: standalone mode (not using charmrun)
Charm++> Running in Multicore mode: 1 threads (PEs)
Converse/Charm++ Commit ID: v7.0.0
Charm++ built without optimization.
Do not use for performance benchmarking (build with --with-production to do so).
Charm++ built with internal error checking enabled.
Do not use for performance benchmarking (build without --enable-error-checking to do so).
CharmLB> Load balancer assumes all CPUs are same.
Charm++> Running on 1 hosts (1 sockets x 2 cores x 1 PUs = 2-way SMP)
Charm++> cpu topology info is gathered in 0.000 seconds.
SpECTRE Build Information:
Version:                      2022.05.05
Compiled on host:             db6353ef9490
Compiled in directory:        /work/build
Source directory is:          /__w/spectre/spectre
Compiled on git branch:       HEAD
Compiled on git revision:     d150e1d
Linked on:                    Thu Jun  2 19:20:24 2022
Filters: "Unit.Domain.CoordinateMaps.Frustum"

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
"Debug.Unit.Domain.CoordinateMaps.Frustum" is a Catch v2.11.1 host application.
Run with -? for options

-------------------------------------------------------------------------------
Unit.Domain.CoordinateMaps.Frustum
-------------------------------------------------------------------------------
/__w/spectre/spectre/tests/Unit/Domain/CoordinateMaps/Test_Frustum.cpp:493
...............................................................................

/__w/spectre/spectre/tests/Unit/Framework/TestingFramework.hpp:149: FAILED:
  CHECK( a == appx(b) )
with expansion:
  0.0 == Approx( 6.4139380956378913 )
with messages:
  Suite for frustum
  Seed is: 195955208 from /__w/spectre/spectre/tests/Unit/Domain/
  CoordinateMaps/Test_Frustum.cpp:27
  lower_x_lower_base := -8.35317526255154519
  lower_y_lower_base := -3.28247238971119959
  upper_x_lower_base := 3.42750338925868103
  upper_y_lower_base := 7.1504708311447569
  lower_x_upper_base := -9.48640560053999948
  lower_y_upper_base := -11.87982197250031646
  upper_x_upper_base := 13.96211903589790637
  upper_y_upper_base := 12.[4110](https://github.com/sxs-collaboration/spectre/runs/6714495282?check_suite_focus=true#step:12:4111)521735189927
  Seed is: 2110194908 from /__w/spectre/spectre/tests/Unit/Helpers/Domain/
  CoordinateMaps/TestMapHelpers.hpp:428
  Test inverse map
  test_point := { 0.0, 0.0, 0.0 }
  /__w/spectre/spectre/tests/Unit/Helpers/Domain/CoordinateMaps/TestMapHelpers.
  hpp:399: test_point == expected_test_point.value()
  a := { 0.0, 0.0, 0.0 }
  b := { 6.[4139](https://github.com/sxs-collaboration/spectre/runs/6714495282?check_suite_focus=true#step:12:4140)380956378913, 32.12285410688970444, 5.58737759528746292 }

/__w/spectre/spectre/tests/Unit/Framework/TestingFramework.hpp:149: FAILED:
  CHECK( a == appx(b) )
with expansion:
  0.0 == Approx( 32.12285410688970444 )
with messages:
  Suite for frustum
  Seed is: 195955208 from /__w/spectre/spectre/tests/Unit/Domain/
  CoordinateMaps/Test_Frustum.cpp:27
  lower_x_lower_base := -8.35317526255154519
  lower_y_lower_base := -3.28247238971119959
  upper_x_lower_base := 3.42750338925868103
  upper_y_lower_base := 7.1504708311447569
  lower_x_upper_base := -9.48640560053999948
  lower_y_upper_base := -11.87982197250031646
  upper_x_upper_base := 13.96211903589790637
  upper_y_upper_base := 12.4110521735189927
  Seed is: 2110194908 from /__w/spectre/spectre/tests/Unit/Helpers/Domain/
  CoordinateMaps/TestMapHelpers.hpp:428
  Test inverse map
  test_point := { 0.0, 0.0, 0.0 }
  /__w/spectre/spectre/tests/Unit/Helpers/Domain/CoordinateMaps/TestMapHelpers.
  hpp:399: test_point == expected_test_point.value()
  a := { 0.0, 0.0, 0.0 }
  b := { 6.4139380956378913, 32.12285410688970444, 5.58737759528746292 }

/__w/spectre/spectre/tests/Unit/Framework/TestingFramework.hpp:149: FAILED:
  CHECK( a == appx(b) )
with expansion:
  0.0 == Approx( 5.58737759528746292 )
with messages:
  Suite for frustum
  Seed is: 195955208 from /__w/spectre/spectre/tests/Unit/Domain/
  CoordinateMaps/Test_Frustum.cpp:27
  lower_x_lower_base := -8.35317526255154519
  lower_y_lower_base := -3.28247238971119959
  upper_x_lower_base := 3.42750338925868103
  upper_y_lower_base := 7.1504708311447569
  lower_x_upper_base := -9.48640560053999948
  lower_y_upper_base := -11.87982197250031646
  upper_x_upper_base := 13.96211903589790637
  upper_y_upper_base := 12.4110521735189927
  Seed is: 2110194908 from /__w/spectre/spectre/tests/Unit/Helpers/Domain/
  CoordinateMaps/TestMapHelpers.hpp:428
  Test inverse map
  test_point := { 0.0, 0.0, 0.0 }
  /__w/spectre/spectre/tests/Unit/Helpers/Domain/CoordinateMaps/TestMapHelpers.
  hpp:399: test_point == expected_test_point.value()
  a := { 0.0, 0.0, 0.0 }
  b := { 6.4139380956378913, 32.12285410688970444, 5.58737759528746292 }
```

There's more to this error, but it's mostly just these same values.

@gsb76 could you look into this?

**Comment by nilsdeppe** [comment]
Another:

```
2022-06-04T16:47:27.6458942Z "Release.Unit.Domain.CoordinateMaps.Frustum" is a Catch v2.11.1 host application.
2022-06-04T16:47:27.6459269Z Run with -? for options
2022-06-04T16:47:27.6459390Z 
2022-06-04T16:47:27.6459635Z -------------------------------------------------------------------------------
2022-06-04T16:47:27.6459940Z Unit.Domain.CoordinateMaps.Frustum
2022-06-04T16:47:27.6460322Z -------------------------------------------------------------------------------
2022-06-04T16:47:27.6460671Z /__w/spectre/spectre/tests/Unit/Domain/CoordinateMaps/Test_Frustum.cpp:493
2022-06-04T16:47:27.6460963Z ...............................................................................
2022-06-04T16:47:27.6461102Z 
2022-06-04T16:47:27.6461248Z /__w/spectre/spectre/tests/Unit/Framework/TestingFramework.hpp:149: FAILED:
2022-06-04T16:47:27.6461512Z   CHECK( a == appx(b) )
2022-06-04T16:47:27.6461705Z with expansion:
2022-06-04T16:47:27.6461884Z   0.16747975149352157
2022-06-04T16:47:27.6462055Z   ==
2022-06-04T16:47:27.6462269Z   Approx( -1.65377850729843456 )
2022-06-04T16:47:27.6462462Z with messages:
2022-06-04T16:47:27.6462646Z   Suite for frustum
2022-06-04T16:47:27.6462886Z   Seed is: 2250086320 from /__w/spectre/spectre/tests/Unit/Domain/
2022-06-04T16:47:27.6463146Z   CoordinateMaps/Test_Frustum.cpp:27
2022-06-04T16:47:27.6463451Z   lower_x_lower_base := -7.12553015234114362
2022-06-04T16:47:27.6463724Z   lower_y_lower_base := -3.81578343644894158
2022-06-04T16:47:27.6463950Z   upper_x_lower_base := 4.22006898697818666
2022-06-04T16:47:27.6464177Z   upper_y_lower_base := 8.67445096160287932
2022-06-04T16:47:27.6464453Z   lower_x_upper_base := -9.96282313151936094
2022-06-04T16:47:27.6464740Z   lower_y_upper_base := -13.43835255603225676
2022-06-04T16:47:27.6464959Z   upper_x_upper_base := 13.48945788347882058
2022-06-04T16:47:27.6465190Z   upper_y_upper_base := 10.59653467984116126
2022-06-04T16:47:27.6465457Z   Seed is: 3771674226 from /__w/spectre/spectre/tests/Unit/Helpers/Domain/
2022-06-04T16:47:27.6465738Z   CoordinateMaps/TestMapHelpers.hpp:428
2022-06-04T16:47:27.6465964Z   Test inverse map
2022-06-04T16:47:27.6466244Z   test_point := { 0.16747975149352157, 0.07839956302669138, -0.
2022-06-04T16:47:27.6466648Z   03467655753088494 }
2022-06-04T16:47:27.6466920Z   /__w/spectre/spectre/tests/Unit/Helpers/Domain/CoordinateMaps/TestMapHelpers.
2022-06-04T16:47:27.6467243Z   hpp:399: test_point == expected_test_point.value()
2022-06-04T16:47:27.6467594Z   a := { 0.16747975149352157, 0.07839956302669138, -0.03467655753088494 }
2022-06-04T16:47:27.6467920Z   b := { -1.65377850729843456, 0.25462168757997455, 4.48189900822158815 }
2022-06-04T16:47:27.6468058Z 
2022-06-04T16:47:27.6468207Z /__w/spectre/spectre/tests/Unit/Framework/TestingFramework.hpp:149: FAILED:
2022-06-04T16:47:27.6468472Z   CHECK( a == appx(b) )
2022-06-04T16:47:27.6468786Z with expansion:
2022-06-04T16:47:27.6468977Z   0.07839956302669138
2022-06-04T16:47:27.6469150Z   ==
2022-06-04T16:47:27.6469315Z   Approx( 0.25462168757997455 )
2022-06-04T16:47:27.6469508Z with messages:
2022-06-04T16:47:27.6469696Z   Suite for frustum
2022-06-04T16:47:27.6469935Z   Seed is: 2250086320 from /__w/spectre/spectre/tests/Unit/Domain/
2022-06-04T16:47:27.6470198Z   CoordinateMaps/Test_Frustum.cpp:27
2022-06-04T16:47:27.6470486Z   lower_x_lower_base := -7.12553015234114362
2022-06-04T16:47:27.6470772Z   lower_y_lower_base := -3.81578343644894158
2022-06-04T16:47:27.6471000Z   upper_x_lower_base := 4.22006898697818666
2022-06-04T16:47:27.6471223Z   upper_y_lower_base := 8.67445096160287932
2022-06-04T16:47:27.6471498Z   lower_x_upper_base := -9.96282313151936094
2022-06-04T16:47:27.6471783Z   lower_y_upper_base := -13.43835255603225676
2022-06-04T16:47:27.6472063Z   upper_x_upper_base := 13.48945788347882058
2022-06-04T16:47:27.6472288Z   upper_y_upper_base := 10.59653467984116126
2022-06-04T16:47:27.6472559Z   Seed is: 3771674226 from /__w/spectre/spectre/tests/Unit/Helpers/Domain/
2022-06-04T16:47:27.6472841Z   CoordinateMaps/TestMapHelpers.hpp:428
2022-06-04T16:47:27.6473066Z   Test inverse map
2022-06-04T16:47:27.6473352Z   test_point := { 0.16747975149352157, 0.07839956302669138, -0.
2022-06-04T16:47:27.6473556Z   03467655753088494 }
2022-06-04T16:47:27.6473825Z   /__w/spectre/spectre/tests/Unit/Helpers/Domain/CoordinateMaps/TestMapHelpers.
2022-06-04T16:47:27.6474140Z   hpp:399: test_point == expected_test_point.value()
2022-06-04T16:47:27.6474480Z   a := { 0.16747975149352157, 0.07839956302669138, -0.03467655753088494 }
2022-06-04T16:47:27.6474801Z   b := { -1.65377850729843456, 0.25462168757997455, 4.48189900822158815 }
2022-06-04T16:47:27.6474923Z 
2022-06-04T16:47:27.6475085Z /__w/spectre/spectre/tests/Unit/Framework/TestingFramework.hpp:149: FAILED:
2022-06-04T16:47:27.6475351Z   CHECK( a == appx(b) )
2022-06-04T16:47:27.6475543Z with expansion:
2022-06-04T16:47:27.6475764Z   -0.03467655753088494
2022-06-04T16:47:27.6475939Z   ==
2022-06-04T16:47:27.6476100Z   Approx( 4.48189900822158815 )
2022-06-04T16:47:27.6476290Z with messages:
2022-06-04T16:47:27.6476472Z   Suite for frustum
2022-06-04T16:47:27.6476711Z   Seed is: 2250086320 from /__w/spectre/spectre/tests/Unit/Domain/
2022-06-04T16:47:27.6476972Z   CoordinateMaps/Test_Frustum.cpp:27
2022-06-04T16:47:27.6477255Z   lower_x_lower_base := -7.12553015234114362
2022-06-04T16:47:27.6477534Z   lower_y_lower_base := -3.81578343644894158
2022-06-04T16:47:27.6477759Z   upper_x_lower_base := 4.22006898697818666
2022-06-04T16:47:27.6477984Z   upper_y_lower_base := 8.67445096160287932
2022-06-04T16:47:27.6478264Z   lower_x_upper_base := -9.96282313151936094
2022-06-04T16:47:27.6478534Z   lower_y_upper_base := -13.43835255603225676
2022-06-04T16:47:27.6478762Z   upper_x_upper_base := 13.48945788347882058
2022-06-04T16:47:27.6478990Z   upper_y_upper_base := 10.59653467984116126
2022-06-04T16:47:27.6479260Z   Seed is: 3771674226 from /__w/spectre/spectre/tests/Unit/Helpers/Domain/
2022-06-04T16:47:27.6479538Z   CoordinateMaps/TestMapHelpers.hpp:428
2022-06-04T16:47:27.6479762Z   Test inverse map
2022-06-04T16:47:27.6480042Z   test_point := { 0.16747975149352157, 0.07839956302669138, -0.
2022-06-04T16:47:27.6480243Z   03467655753088494 }
2022-06-04T16:47:27.6480507Z   /__w/spectre/spectre/tests/Unit/Helpers/Domain/CoordinateMaps/TestMapHelpers.
2022-06-04T16:47:27.6480814Z   hpp:399: test_point == expected_test_point.value()
2022-06-04T16:47:27.6481150Z   a := { 0.16747975149352157, 0.07839956302669138, -0.03467655753088494 }
2022-06-04T16:47:27.6481454Z   b := { -1.65377850729843456, 0.25462168757997455, 4.48189900822158815 }
2022-06-04T16:47:27.6481590Z 
2022-06-04T16:47:27.6481746Z /__w/spectre/spectre/tests/Unit/Framework/TestingFramework.hpp:149: FAILED:
2022-06-04T16:47:27.6482013Z   CHECK( a == appx(b) )
2022-06-04T16:47:27.6482205Z with expansion:
2022-06-04T16:47:27.6482383Z   0.16747975149352157
2022-06-04T16:47:27.6482551Z   ==
2022-06-04T16:47:27.6482842Z   Approx( -1.65377850729843456 )
2022-06-04T16:47:27.6483042Z with messages:
2022-06-04T16:47:27.6483226Z   Suite for frustum
2022-06-04T16:47:27.6483467Z   Seed is: 2250086320 from /__w/spectre/spectre/tests/Unit/Domain/
2022-06-04T16:47:27.6483729Z   CoordinateMaps/Test_Frustum.cpp:27
2022-06-04T16:47:27.6484017Z   lower_x_lower_base := -7.12553015234114362
2022-06-04T16:47:27.6484305Z   lower_y_lower_base := -3.81578343644894158
2022-06-04T16:47:27.6484533Z   upper_x_lower_base := 4.22006898697818666
2022-06-04T16:47:27.6484759Z   upper_y_lower_base := 8.67445096160287932
2022-06-04T16:47:27.6485041Z   lower_x_upper_base := -9.96282313151936094
2022-06-04T16:47:27.6485313Z   lower_y_upper_base := -13.43835255603225676
2022-06-04T16:47:27.6485539Z   upper_x_upper_base := 13.48945788347882058
2022-06-04T16:47:27.6485766Z   upper_y_upper_base := 10.59653467984116126
2022-06-04T16:47:27.6486106Z   Seed is: 3771674226 from /__w/spectre/spectre/tests/Unit/Helpers/Domain/
2022-06-04T16:47:27.6486390Z   CoordinateMaps/TestMapHelpers.hpp:428
2022-06-04T16:47:27.6486614Z   Test inverse map
2022-06-04T16:47:27.6486899Z   test_point := { 0.16747975149352157, 0.07839956302669138, -0.
2022-06-04T16:47:27.6487103Z   03467655753088494 }
2022-06-04T16:47:27.6487372Z   /__w/spectre/spectre/tests/Unit/Helpers/Domain/CoordinateMaps/TestMapHelpers.
2022-06-04T16:47:27.6487681Z   hpp:399: test_point == expected_test_point.value()
2022-06-04T16:47:27.6488020Z   a := { 0.16747975149352157, 0.07839956302669138, -0.03467655753088494 }
2022-06-04T16:47:27.6488323Z   b := { -1.65377850729843456, 0.25462168757997455, 4.48189900822158815 }
2022-06-04T16:47:27.6488458Z 
2022-06-04T16:47:27.6488619Z /__w/spectre/spectre/tests/Unit/Framework/TestingFramework.hpp:149: FAILED:
2022-06-04T16:47:27.6488885Z   CHECK( a == appx(b) )
2022-06-04T16:47:27.6489075Z with expansion:
2022-06-04T16:47:27.6489258Z   0.07839956302669138
2022-06-04T16:47:27.6489429Z   ==
2022-06-04T16:47:27.6489591Z   Approx( 0.25462168757997455 )
2022-06-04T16:47:27.6489783Z with messages:
2022-06-04T16:47:27.6489967Z   Suite for frustum
2022-06-04T16:47:27.6490205Z   Seed is: 2250086320 from /__w/spectre/spectre/tests/Unit/Domain/
2022-06-04T16:47:27.6490466Z   CoordinateMaps/Test_Frustum.cpp:27
2022-06-04T16:47:27.6490749Z   lower_x_lower_base := -7.12553015234114362
2022-06-04T16:47:27.6491030Z   lower_y_lower_base := -3.81578343644894158
2022-06-04T16:47:27.6491253Z   upper_x_lower_base := 4.22006898697818666
2022-06-04T16:47:27.6491477Z   upper_y_lower_base := 8.67445096160287932
2022-06-04T16:47:27.6491872Z   lower_x_upper_base := -9.96282313151936094
2022-06-04T16:47:27.6589308Z   lower_y_upper_base := -13.43835255603225676
2022-06-04T16:47:27.6589563Z   upper_x_upper_base := 13.48945788347882058
2022-06-04T16:47:27.6589798Z   upper_y_upper_base := 10.59653467984116126
2022-06-04T16:47:27.6590081Z   Seed is: 3771674226 from /__w/spectre/spectre/tests/Unit/Helpers/Domain/
2022-06-04T16:47:27.6590366Z   CoordinateMaps/TestMapHelpers.hpp:428
2022-06-04T16:47:27.6590767Z   Test inverse map
2022-06-04T16:47:27.6591073Z   test_point := { 0.16747975149352157, 0.07839956302669138, -0.
2022-06-04T16:47:27.6591266Z   03467655753088494 }
2022-06-04T16:47:27.6591538Z   /__w/spectre/spectre/tests/Unit/Helpers/Domain/CoordinateMaps/TestMapHelpers.
2022-06-04T16:47:27.6591848Z   hpp:399: test_point == expected_test_point.value()
2022-06-04T16:47:27.6592190Z   a := { 0.16747975149352157, 0.07839956302669138, -0.03467655753088494 }
2022-06-04T16:47:27.6592513Z   b := { -1.65377850729843456, 0.25462168757997455, 4.48189900822158815 }
2022-06-04T16:47:27.6592655Z 
2022-06-04T16:47:27.6592814Z /__w/spectre/spectre/tests/Unit/Framework/TestingFramework.hpp:149: FAILED:
2022-06-04T16:47:27.6593067Z   CHECK( a == appx(b) )
2022-06-04T16:47:27.6593259Z with expansion:
2022-06-04T16:47:27.6593481Z   -0.03467655753088494
2022-06-04T16:47:27.6593656Z   ==
2022-06-04T16:47:27.6593832Z   Approx( 4.48189900822158815 )
2022-06-04T16:47:27.6594010Z with messages:
2022-06-04T16:47:27.6594349Z   Suite for frustum
2022-06-04T16:47:27.6594594Z   Seed is: 2250086320 from /__w/spectre/spectre/tests/Unit/Domain/
2022-06-04T16:47:27.6594856Z   CoordinateMaps/Test_Frustum.cpp:27
2022-06-04T16:47:27.6595156Z   lower_x_lower_base := -7.12553015234114362
2022-06-04T16:47:27.6595428Z   lower_y_lower_base := -3.81578343644894158
2022-06-04T16:47:27.6595653Z   upper_x_lower_base := 4.22006898697818666
2022-06-04T16:47:27.6595878Z   upper_y_lower_base := 8.67445096160287932
2022-06-04T16:47:27.6596158Z   lower_x_upper_base := -9.96282313151936094
2022-06-04T16:47:27.6596447Z   lower_y_upper_base := -13.43835255603225676
2022-06-04T16:47:27.6596676Z   upper_x_upper_base := 13.48945788347882058
2022-06-04T16:47:27.6596889Z   upper_y_upper_base := 10.59653467984116126
2022-06-04T16:47:27.6597154Z   Seed is: 3771674226 from /__w/spectre/spectre/tests/Unit/Helpers/Domain/
2022-06-04T16:47:27.6597509Z   CoordinateMaps/TestMapHelpers.hpp:428
2022-06-04T16:47:27.6597737Z   Test inverse map
2022-06-04T16:47:27.6598038Z   test_point := { 0.16747975149352157, 0.07839956302669138, -0.
2022-06-04T16:47:27.6598230Z   03467655753088494 }
2022-06-04T16:47:27.6598497Z   /__w/spectre/spectre/tests/Unit/Helpers/Domain/CoordinateMaps/TestMapHelpers.
2022-06-04T16:47:27.6598805Z   hpp:399: test_point == expected_test_point.value()
2022-06-04T16:47:27.6599147Z   a := { 0.16747975149352157, 0.07839956302669138, -0.03467655753088494 }
2022-06-04T16:47:27.6599466Z   b := { -1.65377850729843456, 0.25462168757997455, 4.48189900822158815 }
2022-06-04T16:47:27.6599604Z 
2022-06-04T16:47:27.6599731Z 0.114 s: Unit.Domain.CoordinateMaps.Frustum
2022-06-04T16:47:27.6599977Z ===============================================================================
2022-06-04T16:47:27.6600210Z test cases:     1 |     0 passed | 1 failed
2022-06-04T16:47:27.6600440Z assertions: 71213 | 71207 passed | 6 failed
2022-06-04T16:47:27.6600577Z 
2022-06-04T16:47:27.6600663Z A catch test has failed.
```

**Comment by nilsdeppe** [comment]
And another...
```
022-06-05T05:16:57.7406295Z 
2022-06-05T05:16:57.7406474Z /__w/spectre/spectre/tests/Unit/Framework/TestingFramework.hpp:149: FAILED:
2022-06-05T05:16:57.7406780Z   CHECK( a == appx(b) )
2022-06-05T05:16:57.7406996Z with expansion:
2022-06-05T05:16:57.7407302Z   0.0 == Approx( -1.57689861020254085 )
2022-06-05T05:16:57.7407526Z with messages:
2022-06-05T05:16:57.7407718Z   Suite for frustum
2022-06-05T05:16:57.7407990Z   Seed is: 2275937136 from /__w/spectre/spectre/tests/Unit/Domain/
2022-06-05T05:16:57.7408287Z   CoordinateMaps/Test_Frustum.cpp:27
2022-06-05T05:16:57.7408622Z   lower_x_lower_base := -8.59639044037647615
2022-06-05T05:16:57.7408947Z   lower_y_lower_base := -3.77937852030115362
2022-06-05T05:16:57.7409186Z   upper_x_lower_base := 3.17168063113010179
2022-06-05T05:16:57.7409439Z   upper_y_lower_base := 6.98982183461436257
2022-06-05T05:16:57.7409756Z   lower_x_upper_base := -10.36489946511750659
2022-06-05T05:16:57.7410081Z   lower_y_upper_base := -13.98372896763795126
2022-06-05T05:16:57.7410338Z   upper_x_upper_base := 12.1214656504653675
2022-06-05T05:16:57.7410578Z   upper_y_upper_base := 13.512673581329679
2022-06-05T05:16:57.7410884Z   Seed is: 1644911340 from /__w/spectre/spectre/tests/Unit/Helpers/Domain/
2022-06-05T05:16:57.7411201Z   CoordinateMaps/TestMapHelpers.hpp:428
2022-06-05T05:16:57.7411455Z   Test inverse map
2022-06-05T05:16:57.7411678Z   test_point := { 0.0, 0.0, 0.0 }
2022-06-05T05:16:57.7411995Z   /__w/spectre/spectre/tests/Unit/Helpers/Domain/CoordinateMaps/TestMapHelpers.
2022-06-05T05:16:57.7412331Z   hpp:399: test_point == expected_test_point.value()
2022-06-05T05:16:57.7412586Z   a := { 0.0, 0.0, 0.0 }
2022-06-05T05:16:57.7412930Z   b := { -1.57689861020254085, 0.08279514304363567, 5.32288828392161051 }
2022-06-05T05:16:57.7413087Z 
2022-06-05T05:16:57.7413265Z /__w/spectre/spectre/tests/Unit/Framework/TestingFramework.hpp:149: FAILED:
2022-06-05T05:16:57.7413562Z   CHECK( a == appx(b) )
2022-06-05T05:16:57.7413769Z with expansion:
2022-06-05T05:16:57.7413991Z   0.0 == Approx( 0.08279514304363567 )
2022-06-05T05:16:57.7414210Z with messages:
2022-06-05T05:16:57.7414416Z   Suite for frustum
2022-06-05T05:16:57.7414687Z   Seed is: 2275937136 from /__w/spectre/spectre/tests/Unit/Domain/
2022-06-05T05:16:57.7414968Z   CoordinateMaps/Test_Frustum.cpp:27
2022-06-05T05:16:57.7415303Z   lower_x_lower_base := -8.59639044037647615
2022-06-05T05:16:57.7415626Z   lower_y_lower_base := -3.77937852030115362
2022-06-05T05:16:57.7415880Z   upper_x_lower_base := 3.17168063113010179
2022-06-05T05:16:57.7416135Z   upper_y_lower_base := 6.98982183461436257
2022-06-05T05:16:57.7416551Z   lower_x_upper_base := -10.36489946511750659
2022-06-05T05:16:57.7416874Z   lower_y_upper_base := -13.98372896763795126
2022-06-05T05:16:57.7417129Z   upper_x_upper_base := 12.1214656504653675
2022-06-05T05:16:57.7417383Z   upper_y_upper_base := 13.512673581329679
2022-06-05T05:16:57.7417680Z   Seed is: 1644911340 from /__w/spectre/spectre/tests/Unit/Helpers/Domain/
2022-06-05T05:16:57.7417995Z   CoordinateMaps/TestMapHelpers.hpp:428
2022-06-05T05:16:57.7418236Z   Test inverse map
2022-06-05T05:16:57.7418461Z   test_point := { 0.0, 0.0, 0.0 }
2022-06-05T05:16:57.7418852Z   /__w/spectre/spectre/tests/Unit/Helpers/Domain/CoordinateMaps/TestMapHelpers.
2022-06-05T05:16:57.7419204Z   hpp:399: test_point == expected_test_point.value()
2022-06-05T05:16:57.7419455Z   a := { 0.0, 0.0, 0.0 }
2022-06-05T05:16:57.7419788Z   b := { -1.57689861020254085, 0.08279514304363567, 5.32288828392161051 }
```

**Comment by knelli2** [comment]
Another

```
209/1328 Test  #298: Unit.Domain.CoordinateMaps.Frustum ..........................................***Failed    0.91 sec
Charm++> No provisioning arguments specified. Running with a single PE.
         Use +auto-provision to fully subscribe resources or +p1 to silence this message.
Charm++: standalone mode (not using charmrun)
Charm++> Running in Multicore mode: 1 threads (PEs)
Converse/Charm++ Commit ID: v7.0.0
Charm++ built without optimization.
Do not use for performance benchmarking (build with --with-production to do so).
Charm++ built with internal error checking enabled.
Do not use for performance benchmarking (build without --enable-error-checking to do so).
CharmLB> Load balancer assumes all CPUs are same.
Charm++> Running on 1 hosts (1 sockets x 2 cores x 1 PUs = 2-way SMP)
Charm++> cpu topology info is gathered in 0.000 seconds.
SpECTRE Build Information:
Version:                      2022.05.05
Compiled on host:             9e7da10b8630
Compiled in directory:        /work/build
Source directory is:          /__w/spectre/spectre
Compiled on git branch:       HEAD
Compiled on git revision:     6760935
Linked on:                    Mon Jun  6 15:20:04 2022
Filters: "Unit.Domain.CoordinateMaps.Frustum"
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
"Debug.Unit.Domain.CoordinateMaps.Frustum" is a Catch v2.11.1 host application.
Run with -? for options
-------------------------------------------------------------------------------
Unit.Domain.CoordinateMaps.Frustum
-------------------------------------------------------------------------------
/__w/spectre/spectre/tests/Unit/Domain/CoordinateMaps/Test_Frustum.cpp:493
...............................................................................
/__w/spectre/spectre/tests/Unit/Framework/TestingFramework.hpp:149: FAILED:
  CHECK( a == appx(b) )
with expansion:
  0.0 == Approx( -1.58855682385840646 )
with messages:
  Suite for frustum
  Seed is: 3716635621 from /__w/spectre/spectre/tests/Unit/Domain/
  CoordinateMaps/Test_Frustum.cpp:27
  lower_x_lower_base := -8.68424202281085478
  lower_y_lower_base := -8.96455375051892034
  upper_x_lower_base := 4.52469483225359781
  upper_y_lower_base := 8.50592394612718117
  lower_x_upper_base := -9.47465122686936745
  lower_y_upper_base := -11.96959567801616942
  upper_x_upper_base := 12.09604010287710096
  upper_y_upper_base := 12.59884811958370854
  Seed is: 1781726698 from /__w/spectre/spectre/tests/Unit/Helpers/Domain/
  CoordinateMaps/TestMapHelpers.hpp:428
  Test inverse map
  test_point := { 0.0, 0.0, 0.0 }
  /__w/spectre/spectre/tests/Unit/Helpers/Domain/CoordinateMaps/TestMapHelpers.
  hpp:399: test_point == expected_test_point.value()
  a := { 0.0, 0.0, 0.0 }
  b := { -1.58855682385840646, -0.00911745159606917, 5.02769[435](https://github.com/sxs-collaboration/spectre/runs/6757959538?check_suite_focus=true#step:12:436)983829638 }
/__w/spectre/spectre/tests/Unit/Framework/TestingFramework.hpp:149: FAILED:
  CHECK( a == appx(b) )
with expansion:
  0.0 == Approx( -0.009117[451](https://github.com/sxs-collaboration/spectre/runs/6757959538?check_suite_focus=true#step:12:452)59606917 )
with messages:
  Suite for frustum
  Seed is: 3716635621 from /__w/spectre/spectre/tests/Unit/Domain/
  CoordinateMaps/Test_Frustum.cpp:27
  lower_x_lower_base := -8.68424202281085478
  lower_y_lower_base := -8.96[455](https://github.com/sxs-collaboration/spectre/runs/6757959538?check_suite_focus=true#step:12:456)375051892034
  upper_x_lower_base := 4.52469483225359781
  upper_y_lower_base := 8.5059239[461](https://github.com/sxs-collaboration/spectre/runs/6757959538?check_suite_focus=true#step:12:462)2718117
  lower_x_upper_base := -9.47465122686936745
  lower_y_upper_base := -11.96959567801616942
  upper_x_upper_base := 12.09604010287710096
  upper_y_upper_base := 12.59884811958370854
  Seed is: 1781726698 from /__w/spectre/spectre/tests/Unit/Helpers/Domain/
  CoordinateMaps/TestMapHelpers.hpp:428
  Test inverse map
  test_point := { 0.0, 0.0, 0.0 }
  /__w/spectre/spectre/tests/Unit/Helpers/Domain/CoordinateMaps/TestMapHelpers.
  hpp:399: test_point == expected_test_point.value()
  a := { 0.0, 0.0, 0.0 }
  b := { -1.58855682385840646, -0.00911745159606917, 5.02769435983829638 }
/__w/spectre/spectre/tests/Unit/Framework/TestingFramework.hpp:149: FAILED:
  CHECK( a == appx(b) )
with expansion:
  0.0 == Approx( 5.02769435983829638 )
with messages:
  Suite for frustum
  Seed is: 3716635621 from /__w/spectre/spectre/tests/Unit/Domain/
  CoordinateMaps/Test_Frustum.cpp:27
  lower_x_lower_base := -8.68424202281085478
  lower_y_lower_base := -8.96455375051892034
  upper_x_lower_base := 4.52469483225359781
  upper_y_lower_base := 8.50592394612718117
  lower_x_upper_base := -9.47[465](https://github.com/sxs-collaboration/spectre/runs/6757959538?check_suite_focus=true#step:12:466)122686936745
  lower_y_upper_base := -11.96959567801616942
  upper_x_upper_base := 12.09604010287710096
  upper_y_upper_base := 12.59884811958370854
  Seed is: 1781726698 from /__w/spectre/spectre/tests/Unit/Helpers/Domain/
  CoordinateMaps/TestMapHelpers.hpp:428
  Test inverse map
  test_point := { 0.0, 0.0, 0.0 }
  /__w/spectre/spectre/tests/Unit/Helpers/Domain/CoordinateMaps/TestMapHelpers.
  hpp:399: test_point == expected_test_point.value()
  a := { 0.0, 0.0, 0.0 }
  b := { -1.58855682385840646, -0.0091174[515](https://github.com/sxs-collaboration/spectre/runs/6757959538?check_suite_focus=true#step:12:516)9606917, 5.02769435983829638 }
```

**Comment by nilsdeppe** [comment]
Fixed in #4034
